### PR TITLE
Channel list extension

### PR DESCRIPTION
**Pull request by AylinKlinge**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 15 June 2021 12:00 UTC
+
+* [Channel](channels.md#channels) list extended. The latest Channels added have codes `854`, `855`, `856`, `857`, `858`, `859`, `860`, `861`, `862`, `863`, `864`, `864`, `865`, `866`, `867`, `868`, `869`.
```
Comment (patrikmiksovsky95) [inline changelog.md:5]: We write there only latest code

Comment (patrikmiksovsky95) [reply changelog.md:5]: @AylinKlinge ping

Comment (AylinKlinge) [reply changelog.md:5]: @patrikmiksovsky95 what do you mean? Ignore all other new codes and mention only one?

Comment (patrikmiksovsky95) [reply changelog.md:5]: @AylinKlinge yes exactly

**Comment by patrikmiksovsky95** [comment]
@AylinKlinge please do it for new format of documentation as new pull request

**Pull request closed without merging**